File: 30-Days-of-Code/Day-0-Hello-World./Rust/Solution.rs
use std::io::{self,BufRead,Write};

fn main() {
    let mut line = String::new();
    let stdin = io::stdin();
    stdin.lock().read_line(&mut line).unwrap();
    println!("Hello, World.");
    
    println!("{}", line);
    //try!(io::stdout().write(line));
}

File: 30-Days-of-Code/Day-0-Hello-World./Go/Solution.go
package main

import (
    "fmt"
    "bufio"
    "os"
)

func main() {
    reader := bufio.NewReader(os.Stdin)
    text, _ := reader.ReadString('\n')
    fmt.Println("Hello, World.")
    fmt.Println(text)
}
